test-java-lib-lombok/src/main/java/com/uber/lombok/LombokDTO.java

```diff
@@ -32,7 +32,9 @@
   "SameNameButDifferent" /* crashes with EP 2.6.0 */,
   "InlineMeInliner" /* crashes with EP 2.7.1 */
 })
-public class LombokBuilderInit {
+public class LombokDTO {
+  // Note: Field initialization is done by Lombok generated code. NullAway issues no error due to
+  // field being uninitialized.
```

Comment:
This might be a slightly clearer wording:
```suggestion
  // Note: Lombok generates a constructor directly into this class that initializes this field, so
  // NullAway does not issue an uninitialized field warning.
```